appimage/Readme.md

```diff
@@ -0,0 +1,24 @@
+AppImage Packaging
```

Comment:
Please name this README.md (caps) for consistency

Reply:
Done